backend/apid/actions/generic.go

```diff
@@ -0,0 +1,23 @@
+package actions
+
+import (
+	storev2 "github.com/sensu/sensu-go/storage"
+	"golang.org/x/net/context"
```

Comment:
This should probably be stdlib context

Reply:
Huh, it seems like we are using `golang.org/x/net/context` in other files within the same package so the editor just picked that. I'll remove all mentions of that pkg in order to switch to the built-in one!